aten/src/ATen/core/op_registration/test_helpers.h

```diff
@@ -34,6 +34,12 @@ inline std::vector<c10::IValue> callOp(const c10::OperatorHandle& op, Args... ar
   return stack;
 }
 
+template<class Result, class... Args>
+inline Result callOpUnboxed(const c10::OperatorHandle& op, c10::TensorTypeId dispatchKey, Args... args) {
```

Comment:
I though the whole point of unboxed call was to figure out the dispatch key from arguments. Do you think it's doable with template magic to collect keys of arguemnts (I recall you had this code somewhere). If not - we can codegen it

Reply:
This is planned but a future PR. There's a mismatch between c10 and aten currently because c10 chooses "the first tensor argument" to dispatch on while aten looks for "the tensor argument named 'self'". The plan is to add this 'self' functionality to c10 and then use it.